### ISSUE TITLE
wondeful work, but a dumb question

### ISSUE THREAD

**Issue by jinfagang** [apchenstu/TensoRF#27]
TensoRF looks like small and very fast traning, is that possible to make the final RGBD result merged into a single point cloud? If so, how? that would be very helpful if can generates a final point cloud.

**Comment by apchenstu** [comment]
Yeah, point cloud sometimes is really useful, actually, you could convert the depth maps and the color result into point could in a simple way from [here](https://github.com/apchenstu/TensoRF/blob/main/renderer.py#L44). You could convert them into point cloud by p = ray_o + depth*ray_d.

**Comment by UPstartDeveloper** [comment]
Hi @jinfagang 

I was planning to try this out too - as another option, it looks like it is possible to visualize a point cloud directly from RGBD scans if you use `open3d`: [docs link](http://www.open3d.org/docs/release/tutorial/geometry/rgbd_image.html). 

I reckon the only pre-requisite to this approach is we would need the depth map to be a grayscale image, and saved separately from the `rgb_map`.
